support.md

```diff
@@ -6,6 +6,28 @@ title: Support
 # Support
 ------------------------------
 
+## Community Guidelines
+
+We have a variety of communication tools that the Foreman Community uses to get
+work done. The guidelines below apply in general when interacting with other
+Foreman users & developers:
+
+* Be nice: Treat people with respect and consideration.
+* Be helpful: Be patient with new people and be willing to jump in to answer questions.
+* Be patient: People often have many other things to do, so allow time for your question to be answered
+* Stay calm: Give people the benefit of the doubt and try not to let emotions get out of control.
```

Comment:
I'm not a native English speaker, but I'd rather like to see something along the lines of "When in doubt, assume the best intentions and do not let emotions get out of control."